package.json

```diff
@@ -23,31 +23,24 @@
   "bugs": {
     "url": "https://github.com/kuzzleio/kuzzle-plugin-auth-github/issues"
   },
-  "pluginInfo": {
-    "defaultConfig": {
-      "strategies": {}
-    }
-  },
   "license": "Apache-2.0",
   "dependencies": {
     "passport": "0.3.2",
     "passport-oauth2": "1.x.x",
     "passport-oauth2-refresh": "1.0.0",
-    "q": "2.0.3"
+    "bluebird": "2.0.3"
```

Comment:
`bluebird` is not needed. At most it is a devDepencency, but you can also remove it entirely (see below).